src/test/java/com/google/enterprise/cloudsearch/sharepoint/SharePointRepositoryTest.java

```diff
@@ -1815,6 +1817,376 @@ public void testGetDocInvalidJsonPayload() throws IOException {
     assertEquals(ApiOperations.deleteItem("item-invalid-payload"), actual);
   }
 
+  @Test
+  public void getDoc_includeExcludeFilter_siteCollection_include() throws IOException {
+    getDoc_includeExcludeFilter_siteCollection("INCLUDE");
```

Comment:
I don't have a strong feeling about this, but we've been using JUnitParams in newer code.